ros2node/test/test_node.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from ros2node.api import annotate_duplicate_nodes
+from ros2node.api import NodeName
 from ros2node.api import parse_node_name
```

Comment:
Is this the right order? I think `N` < `a` in ASCII

Reply:
It is the right order, flake8-import-order does case-insensitive alphasort

Reply:
works for me